homeassistant/components/fritzbox/sensor.py

```diff
@@ -54,6 +55,7 @@ class FritzSensorEntityDescription(
         native_unit_of_measurement=TEMP_CELSIUS,
         device_class=DEVICE_CLASS_TEMPERATURE,
         state_class=STATE_CLASS_MEASUREMENT,
+        entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
```

Comment:
I would consider this to be part of the primary functions.

Reply:
Not in this case, since this is not the room temperature, but the temperature of a smartplug device, those IMO a diagnostic sensor (eq to see if the device overheats)

Reply:
Ah ok.